src/service/repo/anonymousMessageRepo.ts

```diff
@@ -52,8 +54,13 @@ async function checkIsTargetAllowingAnonDM(targetUserId: string) {
   }
 }
 
-async function sendAnonymousMessage(channelId: string, message: string, replyMessageId?: string) {
-  let payload: SendPayloadType = {channelId, message};
+async function sendAnonymousMessage(
+  channelId: string,
+  message: string,
+  attachments: any,
+  replyMessageId?: string
+) {
+  let payload: SendPayloadType = {channelId, message, attachments};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `sendAnonymousMessage` now has an additional parameter `attachments`. This change might break existing calls to this function elsewhere in your codebase. Make sure all calls to this function are updated accordingly. Also, consider making `attachments` an optional parameter to maintain backward compatibility.
```diff
- async function sendAnonymousMessage(
-   channelId: string,
-   message: string,
-   attachments: any,
-   replyMessageId?: string
- ) {
+ async function sendAnonymousMessage(
+   channelId: string,
+   message: string,
+   replyMessageId?: string,
+   attachments?: AttachmentType[]
+ ) {
    let payload: SendPayloadType = {channelId, message, attachments};
    if (replyMessageId) {
      payload = {...payload, messageType: 'reply', replyMessageId};
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->